djangoproject/static/js/mod/search-key.js

```diff
@@ -11,10 +11,15 @@ define([
         init: function(){
             var self = this;
             $(document).ready(function () {
-                $(window).keypress(function(e) {
-                    if ($('input:focus, textarea:focus').length === 0 &&
-                            e.which === 47) {  // The slash is 47.
-                        self.search_form.find('input').focus().select();
+                var search_form_input = self.search_form.find('input');
+                var raw_placeholder = search_form_input.attr('placeholder');
+                var kbd_shortcut_suffix = "Ctrl + K";
+                if (navigator.userAgent.indexOf("Mac") !== -1) kbd_shortcut_suffix = "⌘ + K";
+                search_form_input.attr('placeholder', `${raw_placeholder} (${kbd_shortcut_suffix})`);
```

Comment:
It's more idiomatic in JavaScript to use the ternary operator for such conditional assignments:

```suggestion
                const shortcut = (navigator.userAgent.indexOf("Mac") !== -1) ? "⌘ + K" :  "Ctrl + K";
                search_form_input.attr('placeholder', `${raw_placeholder} (${kbd_shortcut_suffix})`);
```

This also allows us to use `const` to declare that the variable will not change.

Also there's no need to use a verbose variable name when the variable has a single use on the next line 😉